start_cerberus.py

```diff
@@ -138,8 +139,18 @@ def main(cfg):
             logging.info("Detailed inspection of failed components has been enabled")
             inspect.delete_inspect_directory()
 
-        # get list of all master nodes to verify scheduling
-        master_nodes = kubecli.list_nodes("node-role.kubernetes.io/master")
+        # get list of all master nodes with provided labels in the config
+        master_label = watch_master_schedulable["label"]
```

Comment:
Maybe scrape the master label from the config only if watch_master_schedulable is enabled? Thoughts?